inclusive-design/wecount.inclusivedesign.ca
# feat: clicking a filter header on resources page closes/opens the corresponding filter panel

## Description

Clicking a filter header on resources page closes/opens the pane.

## Steps to test

1. Go to the resources page;
2. Hover on one filter header, hover style should be applied to the header;
3. Click the header, the corresponding filter panel should open/close with each click;
4. Click the expand/collapse button in the header. Make sure it behaves correctly.

**Expected behavior:** 
See above.
